projects/plugins/jetpack/_inc/client/recommendations/feature-summary/style.scss

```diff
@@ -14,6 +14,33 @@
 	&:last-child {
 		border-bottom: none;
 	}
+
+	&.is-primary {
+		$spacing: 0.5rem;
+
+		padding-top: $spacing;
+		padding-bottom: $spacing;
```

Comment:
`padding: $spacing 0` is a bit more concise in this case, and since the `$spacing` variable would only be used once you could remove it entirely and just use `padding: 0.5rem 0`. This is just what I would do and feel free to let me know if you had a good reason for doing it this way 😄

Reply:
The `is-primary` class is a modifying class. I haven't used the shorthand for padding as I wanted to restrain from overwriting horizontal padding when unnecessary - to make sure it will still inherit values from the parent if we'll e.g. decide to change overall styling a bit.

Reply:
Ah ok yes, this makes sense, thanks!